bindings/nodejs/lib/types/block/context_input/context-input.ts

```diff
@@ -11,17 +11,17 @@ enum ContextInputType {
      */
     COMMITMENT = 0,
     /**
-     * The context input type of a `BlockIssuanceCreditContextInput`.
+     * The context input kind of a `BlockIssuanceCreditContextInput`.
```

Comment:
We don't really have an issue in TS with type being a reserved keyword, not sure I'd change everything, even more since the type is called ContextInputType

Reply:
accidental from rebase, fixed